FEM/rf_pcs.cpp

```diff
@@ -3371,6 +3375,10 @@ void CRFProcess::VariableDynamics()
 	pcs_secondary_function_name[pcs_number_of_secondary_nvals] = "STRAIN_PLS";
 	pcs_secondary_function_unit[pcs_number_of_secondary_nvals] = "-";
 	pcs_secondary_function_timelevel[pcs_number_of_secondary_nvals] = 1;
+	pcs_number_of_secondary_nvals++;
```

Comment:
Makes sense. I'm not sure whether a material model-specific bool is the best way to go about this or something else. I'll pass by your office tomorrow.